### ISSUE TITLE
[$10][MSFT-58] [Registration] Show Button

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2774]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Inspect Element
3. Observe the

**Expected Result**
Expected contrast ratio of 4.5:1. 

**Actual Result**
Element has insufficient color contrast of 1.37 (foreground color: #FFFFFF, background color: ##DCDCE0, font size: small.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.4.1 Use of Color
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096312 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096312 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu this has been taken care in one of my PR, hence marking ready

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096312 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu @codeMinter this issue https://github.com/topcoder-platform/community-app/issues/2868 PR solves it

and PR is  appirio-tech/accounts-app#248

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2774 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2774 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2774 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2774 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097499 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097500 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097500 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Verified color code from https://github.com/topcoder-platform/community-app/issues/2775

**Comment by nkumar-topcoder** [comment]
This has common fix from issue #2868. Will update once 2868 is deployed

**Comment by nkumar-topcoder** [comment]
@mishacucicea #2868, #2775 and #2774 all  fix overlap with similar files, earlier merges failed. Can you please take care of this. The PR are 248, 278.

**Comment by tosha5252** [comment]
referenced #2775...#5d5d66 is the correct color for the button

Windows 10 env
Browser: Chrome

**Comment by tosha5252** [comment]
referenced #2775...color is correct for the button

Windows 10 env
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097500<br/><br/>```This is an automated message for crazyk via Topcoder X```
